### ISSUE TITLE
De-wed from Spotify

### ISSUE THREAD

**Issue by john-bonachon** [jaedb/Iris#65]
I know this is kind of ankward, but it would be great to use Iris without spotify for those who doesn't have Premium accounts.

I've seen it works with another backends, i.e. Mopidy-Youtube, as its results are properly shown in Iris.

Is there any easy way (for you or us) to disable Spotify results from Iris?

That would make Iris a more general frontend and in my opinion the **best** around there.

**Comment by john-bonachon** [comment]
I don't know about React, so here is a very horrible hack,

```css
nav section:nth-child(2) {
  display: none;
}

.search-result-sections {
  display: none;
}
```

```js
setInterval(function(){
  $('.list-item.track .fa-spotify').parent().hide();
}, 200);
```

**Comment by jaedb** [comment]
A lot of Iris functionality requires access to the Spotify HTTP API. This API [generally] requires authentication tokens, which can only be generated from a genuine Spotify user account. Note: this does not need to be a Spotify Premum account.

To use _Mopidy-Spotify_ (the backend that lets us play Spotify tracks), you will need a Spotify Premium account.

Due to this relationship between browsing music and playing music, I don't currently plan to de-wed Iris from Spotify. It is simply too involved a piece of work to warrant for what is a shrinking audience. If it does become highly sought-after, then it may be looked at in the future.

**Comment by john-bonachon** [comment]
I really appreciate your work. Iris was totally made to shine with Spotify and should maintain its concept.

I just think that there could be an on/off switch somewhat forgotten in the Settings to hide Spotify browsing and results (if this doesn't require several code changes). 

Because I believe your app is the most beautiful Mopidy http client even without Spotify neither the need to reimplement the same features for local media.. just Search and Now Playing queue.

**Comment by john-bonachon** [comment]
Ok, got it. 
Meanwhile, could I ask you a better way to replace my ugly hack in my local installation? I was wondering how to hide spotify results rows.

To avoid using setInterval() like above, I've tried doing this, but didn't work (surely because I don't know how React works)
```js
$('body').on('show', '.list-item.track .fa-spotify', function() { 
   $(e.target).parent().hide() 
})
```
or
```js
$('body').on('DOMNodeInserted', '.list-item.track .fa-spotify', function(e) { 
   $(e.target).parent().hide() 
})
```

**Comment by jaedb** [comment]
Resolved in [`2.13.6`](https://github.com/jaedb/Iris/releases/tag/2.13.6)
